boards/frdm-kw41z/include/periph_conf.h

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright (C) 2017 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     board_frdm-kw41z
+ * @{
+ *
+ * @file
+ * @name        Peripheral MCU configuration for the FRDM-KW41Z
+ *
+ * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/**
+ * @name Clock system configuration
+ * @{
+ */
+static const clock_config_t clock_config = {
+    /*
+     * This configuration results in the system running directly from the RF
+     * module clock with the following clock frequencies:
+     * Core:  32 MHz
+     * Bus:   16 MHz
+     * Flash: 16 MHz
+     */
+    .clkdiv1 = SIM_CLKDIV1_OUTDIV1(0) | SIM_CLKDIV1_OUTDIV4(1),
+    .default_mode = KINETIS_MCG_MODE_FEI,
+    /* The crystal connected to RSIM OSC is 32 MHz */
+    .erc_range = KINETIS_MCG_ERC_RANGE_VERY_HIGH,
+    .fcrdiv = 0, /* Fast IRC divide by 1 => 4 MHz */
+    .oscsel = 0, /* Use RSIM for external clock */
+    .clc = 0, /* no load cap configuration */
+    .fll_frdiv = 0b101, /* Divide by 1024 */
+    .fll_factor_fei = KINETIS_MCG_FLL_FACTOR_1464, /* FLL freq = 48 MHz */
+    .fll_factor_fee = KINETIS_MCG_FLL_FACTOR_1280, /* FLL freq = 40 MHz */
+    .enable_oscillator = true, /* Use RF module oscillator */
+    .select_fast_irc = true,
+    .enable_mcgirclk = true, /* Used for LPUART clocking */
+};
```

Comment:
Can the values of this struct be aligned for readability?

Reply:
addressed